source/builders.txt

```diff
@@ -0,0 +1,110 @@
+.. _kotlin-sync-builders:
+
+================
+Use Builders API
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. .. toctree::
+
+..    /fundamentals/builders/aggregates
+..    /fundamentals/builders/filters
+..    /fundamentals/builders/indexes
+..    /fundamentals/builders/projections
+..    /fundamentals/builders/sort
+..    /fundamentals/builders/updates
+
+Overview
+--------
+
+This page describes how to use the builders API and demonstrates the utility provided by
+builder classes.
+
+The {+driver-short+} provides type-safe classes and methods that enable developers to
+efficiently build queries and aggregations.
```

Comment:
```suggestion
The {+driver-short+} provides type-safe builders classes and methods that enable developers to
efficiently build queries and aggregations. To learn more about the builders pattern, see `Type-safe builders <https://kotlinlang.org/docs/type-safe-builders.html>`__ in the kotlin docs
```
Use the source constants ^ for kotlin and the link

Reply:
Great idea to link out here, much obliged.